web/views.py

```
@@ -296,6 +297,25 @@ def index(request):
     return render(request, "index.html", context)
 
 
+@login_required
+def submit_team_proof(request, team_goal_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `submit_team_proof`

(ANN201)